Documentation/ApiOverview/PasswordHashing/Index.rst

```diff
@@ -130,21 +131,21 @@ Configuration
 Configuration of password hashing is done by TYPO3 automatically and
 administrators usually do not need to worry about details too much: The
 installation process will configure the best available hash algorithm by default.
-This is usually Argon2i. Only if the PHP build is incomplete, some less secure
+This is at the moment Argon2i. Only if the PHP build is incomplete, some less secure
```

Comment:
That seems to be not really true ("the best hash algorithm"): In v13 the default hashing algorithm will be argon2id:
https://review.typo3.org/c/Packages/TYPO3.CMS/+/82865

Reply:
How should we replace it or can we link somewhere where the current default can be determined?

Reply:
It is configured in EXT:core/Configuration/DefaultConfiguration.php.

Maybe "one of the best". Argon2i is not the best now.

Background:
https://crypto.stackexchange.com/questions/72416/when-to-use-argon2i-vs-argon2d-vs-argon2id#72437